src/routes/v1/sponsor.route.js

```diff
@@ -8,6 +8,7 @@ router.get('/allsponsors', sponsorController.getAllSponsors);
 router.put('/editsponsor', sponsorController.editSponsor);  // 
 router.delete('/deletesponsor', sponsorController.deleteSponsor);  // 
 router.get('/getsponsorbyid', sponsorController.getSponsorById);
+router.get('/getsponsorbyid1', sponsorController.getSponsorById);
```

Comment:
@sachin WE have already this module beofre , Just reuse that componennt, no need to create 

Can u verify if its work with these browsers 
Can you make sure to handle errors 
Can you use diffferent method as current method is inefficient

Reply:
@ShaivyT changed code sa per your suggetstion

Testd it, it works on all broswers 
Handle errors with  try /catch 

Using .find as its more efficeient